domain/src/main/assets/fractions_exploration1.json

```diff
@@ -3933,7 +3933,7 @@
               "param_changes": [],
               "feedback": {
                 "content_id": "feedback_4",
-                "html": "<p>No, that's not right.</p><p>It looks like you've forgotten what the numerator and denominator represent. You should probably try the <oppia-noninteractive-link text-with-value=\"&amp;quot;refresher lesson&amp;quot;\" url-with-value=\"&amp;quot;https://www.oppia.org/explore/xHwiGtvK2N4L?parent=MjZzEVOG47_1&amp;quot;\"></oppia-noninteractive-link> to refresh your memory.<br></p>"
+                "html": "<p>No, that's not right.</p><p>It looks like you've forgotten what the numerator and denominator represent. You should probably try the <oppia-concept-card-link skill-id=\"UxTGIJqaHMLa\">refresher lesson</oppia-concept-card-link> to refresh your memory.<br></p>"
```

Comment:
I think the text should be: Take a look at this [concept card] to refresh your memory.

Reply:
Done.